modules/claims_api/lib/evss_service/base.rb

```diff
@@ -18,14 +18,19 @@ def initialize(request = nil)
       def submit(claim, data)
         @auth_headers = claim.auth_headers
 
-        client.post('submit', data).body
+        begin
+          client.post('submit', data).body
+        rescue => e
+          ClaimsApi::Logger.log('526', detail: "EVSS CONTAINER submit error: #{e}")
+        end
       end
```

Comment:
@FonzMP  (same note as above, just wanted to address previous comments) This will _not_ return a 500 when triggered.  It allows the process to keep going if the submit fails by just logging out in that failure and then allowing the process to continue.  That _seems_ to be inline with what was wanted for this process, but if not let me know and I can obviously remove it easily enough.